Configuration/Settings.Neos.yaml

```diff
@@ -13,3 +13,6 @@ Neos:
         finisherPresets:
           'FormatD.HubSpot:HubSpotFormFinisher':
             implementationClassName: FormatD\HubSpot\Form\Finishers\HubSpotFormFinisher
+  Fusion:
+    defaultContext:
+      HFile: 'FormatD\HubSpot\Eel\HubspotFileHelper'
```

Comment:
Is this really needed? The helper is only used during mapping of fields right?

Reply:
I tested it, you're right, it wasn't needed.